arts-university-bournemouth.csl

```diff
@@ -0,0 +1,254 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB">
+  <info>
+    <title>Arts University Bournemouth</title>
+    <title-short>AUB</title-short>
+    <id>http://www.zotero.org/styles/arts-university-bournemouth</id>
+    <link href="http://www.zotero.org/styles/arts-university-bournemouth" rel="self"/>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link href="http://www.zotero.org/styles/the-open-university-harvard" rel="template"/>
+    <link href="http://www.aublibrary.info/index.php/referencing" rel="documentation"/>
+    <author>
+      <name>Duncan Belk</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
+    <updated>2017-04-24T10:20:30+00:00</updated>
+  </info>
+  <macro name="editor">
+    <names variable="editor" delimiter=", ">
+      <name and="text" initialize-with=". " delimiter=", " sort-separator=", " name-as-sort-order="all"/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+ <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="text" sort-separator=", " delimiter=", " delimiter-precedes-last="never" form="long" initialize-with=". "/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <choose>
+          <if type="webpage">
+            <text variable="container-title"/>
+            <text variable="URL"/>
```

Comment:
Since you're already adding the URL through the access macro, why not leave it out here?